cogs/tags.py

```diff
@@ -107,115 +170,338 @@ async def all(self, ctx: commands.Context):
             ctx.command.reset_cooldown(ctx)
             return await ctx.send("Could not dm you...", delete_after=10)
 
-        pager = commands.Paginator()
+        async def send_tags():
+            pager = commands.Paginator()
 
-        for record in records:
-            pager.add_line(line=record["name"])
+            for record in records:
+                pager.add_line(line=record["name"])
 
-        for page in pager.pages:
-            await asyncio.sleep(1)
-            await ctx.author.send(page)
+            for page in pager.pages:
+                await asyncio.sleep(1)
+                await ctx.author.send(page)
+
+        asyncio.create_task(send_tags())
 
-        await ctx.send("Tags sent in DMs.")
+        await ctx.send("Tags are being sent in DMs.")
 
     @tag.command()
     @is_engineer_check()
     async def edit(self, ctx, name: lambda inp: inp.lower(), *, text: str):
         """Edit a tag"""
+        name = await commands.clean_content().convert(ctx=ctx, argument=name)
         text = await commands.clean_content().convert(ctx=ctx, argument=text)
```

Comment:
I think `name: commands.clean_content, *, text: commands.clean_content` then `name = name.lower()` would be clearer than invoking the converter for clean content.